src/components/ContinuationIn/UnlimitedLiabilityCorporationInformation.vue

```diff
@@ -0,0 +1,258 @@
+<template>
+  <div
+    v-if="getExistingBusinessInfo"
```

Comment:
Safety check on this component so we don't try to de-reference this object (ie, for computeds) if this object doesn't exist.